### PR TITLE
Removed mobile menu, deleted most desktop menu options

### PR DESCRIPTION
**Pull request by junthehacker**
Removed mobile menu, since we don't really have any functionality that works on mobile lol.
Deleted most desktop menu options, leaves only import image and delete.

### CODE REVIEW

**Comment by junthehacker** [comment]
@hughdani 
I have removed most useless items.
but i kept the import image one on desktop, as I think it is very useful. (chatroom image edit only works if you send the image to chatroom first, but for import, you can edit the image first without sending the original one).

File: public/app/views/ICE/src/method-draw.js [hunk 1]
```diff
@@ -3495,6 +3515,10 @@ var SOTP = 0;
           {sel:'#tool_save', fn: function() { editingsource ? saveSourceEditor(): clickSave() }, evt: 'mouseup', key: [modKey + 'S', true]},
           {sel:'#tool_export', fn: clickExport, evt: 'mouseup'},
           {sel:'#tool_open', fn: clickOpen, evt: 'mouseup'},
+          {sel:'#tool_help', fn: clickHelp, evt: 'mouseup'},
+          {sel:'#tool_about', fn: clickAbout, evt: 'mouseup'},
+          {sel:'.tool_import', fn: clickImport, evt: 'mousedown'},
```
Comment (hughdani) [inline public/app/views/ICE/src/method-draw.js]: Could you have meant to refer .tool_import and .tool_import_mobile by id instead (#)?

**Comment by junthehacker** [comment]
No, should be class
We had a problem where there are duplicate IDs for desktop and mobile causing errors.

Sent from my iPhone

On Mar 10, 2018, at 7:28 PM, Daniel Hugh <notifications@github.com<mailto:notifications@github.com>> wrote:


@hughdani commented on this pull request.

________________________________

In public/app/views/ICE/src/method-draw.js<https://github.com/bestlkh/ice-MC2/pull/308#discussion_r173639370>:

> @@ -3495,6 +3515,10 @@ var SOTP = 0;
           {sel:'#tool_save', fn: function() { editingsource ? saveSourceEditor(): clickSave() }, evt: 'mouseup', key: [modKey + 'S', true]},
           {sel:'#tool_export', fn: clickExport, evt: 'mouseup'},
           {sel:'#tool_open', fn: clickOpen, evt: 'mouseup'},
+          {sel:'#tool_help', fn: clickHelp, evt: 'mouseup'},
+          {sel:'#tool_about', fn: clickAbout, evt: 'mouseup'},
+          {sel:'.tool_import', fn: clickImport, evt: 'mousedown'},


Could you have meant to refer .tool_import and .tool_import_mobile by id instead (#)?

**Comment by hughdani** [comment]
I see, looks like its a mess. Some old styles and jQuery references need cleaning up. I will be looking into refactoring responsiveness for the chatroom soon.

**Comment by junthehacker** [comment]
@hughdani 
wondering what still needs to changed in this PR?
this is only for removing menu items. nothing else.

**Comment by junthehacker** [comment]
lmfao

**Comment by junthehacker** [comment]
@hughdani fixed.
turns out i cannot remove the edit menu items.

**Comment by hughdani** [comment]
These are the remaining issues from before:

> - (UX) importing image from the editor spawns the image near the bottom right of the canvas which may be inconvenient 

It actually spawns in the middle of the canvas. So, a small enough image might not be visible given the device size.

> - (UX) large images imported from the editor keep their size and could possibly take up the entire canvas

Some images such as those generated from a phone have large dimensions. Consequentially, resize handlers will not be visible unless they search for it.

> - (styles) after importing an image and deleting it with the 'X' icon, the font color becomes fainter suggesting that it is disabled when in actuality still works. I have not figured out if it is possible to revert it back to its original colour afterwards.

This is mainly user experience related.

New issues:

- (bug) Right after joining the chatroom, the undo button from the menu can be activated. If you click it, you will be able to drag around some canvas layer (#canvas_background). If this is removed, editor functions such as Send as Image and the Symbol Keyboard no longer work.

**Comment by junthehacker** [comment]
@hughdani 
Import image has nothing to do with this ticket, so will not fix.
Hopefully i have fixed the import image menu item grey out problem.
I have hidden undo/redo menu items, as they do not work properly. can't remove them, so just added `display: none`.

**Comment by hughdani** [comment]
Looks good.

**Pull request merged**